src/pages/WorkspaceDetails/WorkspaceDetails.test.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import { axe, toHaveNoViolations } from 'jest-axe';
+import WorkspaceDetailsPage from './WorkspaceDetails';
+expect.extend(toHaveNoViolations);
```

Comment:
@kdoberst we can remove this since we put it in the JEST setup

Reply:
Actually, we didn't add this to the test setup.  That was the SDK repo.   No worries - I'll set it up in HAC Infra and add it as a separate commit.